xdsl/builder.py

```diff
@@ -24,11 +34,21 @@ class Builder:
     Operations will be inserted in this block.
     """
 
+    _local: ClassVar[_ImplicitBuilders] = _ImplicitBuilders()
```

Comment:
I would probably rename this field to something else.
Like `_implicit_builder_stack` or something?

Reply:
Yeah your name is better. I'm happy to take other suggestions also.